3.17.0 RC2: crash while posting comment

### Steps to reproduce
1. open the detail view of a file
2. fill in a comment
3. post it, app will crash


************ CAUSE OF ERROR ************

java.lang.RuntimeException: An error occurred while executing doInBackground()
at android.os.AsyncTask$4.done(AsyncTask.java:415)
at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:383)
at java.util.concurrent.FutureTask.setException(FutureTask.java:252)
at java.util.concurrent.FutureTask.run(FutureTask.java:271)
at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:305)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
at java.lang.Thread.run(Thread.java:923)
Caused by: java.lang.IllegalArgumentException: Trying to execute a sync operation without storage provider! Please use execute(OwnCloudClient client, FileDataStorageManager storageManager) instead
at com.owncloud.android.operations.common.SyncOperation.execute(SyncOperation.java:154)
at com.owncloud.android.ui.fragment.FileDetailActivitiesFragment$SubmitCommentTask.doInBackground(FileDetailActivitiesFragment.java:479)
at com.owncloud.android.ui.fragment.FileDetailActivitiesFragment$SubmitCommentTask.doInBackground(FileDetailActivitiesFragment.java:460)
at android.os.AsyncTask$3.call(AsyncTask.java:394)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
... 4 more

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30170052
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: OnePlus
Device: OnePlus7T
Model: HD1903
Id: RKQ1.201022.002
Product: OnePlus7T_EEA

************ FIRMWARE ************
SDK: 30
Release: 11
Incremental: 2105071636

--- a/src/main/java/com/owncloud/android/operations/CommentFileOperation.java
+++ b/src/main/java/com/owncloud/android/operations/CommentFileOperation.java
@@ -27,7 +27,6 @@ import com.owncloud.android.lib.common.operations.RemoteOperationResult;
 import com.owncloud.android.lib.common.utils.Log_OC;
 import com.owncloud.android.lib.resources.comments.CommentFileRemoteOperation;
 
-
 /**
  * Comment file
  */

